src/lib.rs

```diff
@@ -12,61 +14,176 @@ pub struct LinkDrop {
     pub accounts: Map<PublicKey, Balance>,
 }
 
-const ACCESS_KEY_ALLOWANCE: u128 = 100_000_000_000_000_000_000;
+/// Access key allowance for linkdrop keys.
+const ACCESS_KEY_ALLOWANCE: u128 = 1_000_000_000_000_000_000_000;
+
+/// Gas attached to the callback from account creation.
+pub const ON_CREATE_ACCOUNT_CALLBACK_GAS: u64 = 100_000_000_000_000;
```

Comment:
I've started to use more conservative gas values. Usually about `20_000_000_000_000` per execution. For example if you have to do a transfer from the callback, then add another `20_000_000_000_000`, so the total of `40_000_000_000_000`.
This helps to fit under the default attached gas form the near-shell and nearlib, which is `100_000_000_000_000`